e2e-tests/cypress-reporter-config.json

```diff
@@ -0,0 +1,12 @@
+{
+  "reporterEnabled": "mocha-junit-reporter, mochawesome",
+  "mochaJunitReporterReporterOptions": {
+    "mochaFile": "cypress/results/result_[hash].xml"
```

Comment:
Burde vel ses i sammenheng med respons i andre PR

Reply:
Mm. Men samtidig henger denne sammen med opprettelsen av en HTML-rapport (se yarn cypress:html-testreport i siste commit)

Reply:
Lar denne stå enn så lenge for å kunne generere en HTML-testrapport. Mulig det blir endringer etter løsning på #257